recipe/gen_patch_json.py

```diff
@@ -455,6 +455,15 @@ def _gen_new_index(repodata, subdir):
             if new_constrains != record.get('constrains', ()):
                 record['constrains'] = new_constrains
 
+        # distributed <2.11.0 does not work with msgpack-python >=1.0
+        # newer versions of distributed require at least msgpack-python >=0.6.0
+        # so we can fix cases where msgpack-python is unbounded
+        # https://github.com/conda-forge/distributed-feedstock/pull/114
+        if record_name == 'distributed'
+            if 'msgpack-python' in record['depends']:
+                i = record['depends'].index('msgpack-python')
+                record['depends'][i] = 'msgpack-python <1.0.0'
```

Comment:
do you also want `>=0.6.0` here too?

Reply:
I'm not sure, the minimum version for 2.11.0 is 0.6.0 but ealier version may have been compatible with older msgpack-python versions.

@jakirkham do you have insight into this?

Reply:
Huh? I mean the minimum version of msgpack of 0.6.0.

Reply:
> So I don't think we need this for older versions

Agreed. Older versions of `distributed` work fine with pre-0.6.0 versions of msgpack. We set the minimum version of msgpack to 0.6.0 because that's when a certain keyword argument was added that we needed to fix the msgpack 1.0 compatibility issues

Reply:
ohhhhhh I see. OK. Then LGTM!

Reply:
> Huh? I mean the minimum version of msgpack of 0.6.0.

Sorry if that's confusing. The risk of reviewing PRs during a conference. 😉 Hopefully James clarified this, but please let us know if you have more questions. 🙂

Reply:
Questions for hotfixes: Once a new version of distributed compatible with msgpack 1.0 is released, does that have any chance to re-pin it to <1.0 when re-ran ?

Reply:
Because of line 463, this fixing is only applied to distributed versions with a dependency of msgpack-python without any version constraints. If there's a new version of distributed with a requirement like
```
- msgpack-python
```
instead of
```
- msgpack-python >=0.6
```
then, yes.

Reply:
Ah ! Yes, thank, for whatever reason I thought that `record['depends']` would be a dict with `msgpack-python` a key. Makes sens.